### PR TITLE
Created the inspect element for right click including the correct locations for the console

### PR DESCRIPTION
**Pull request by bobvanluijt**
Right click inspect element, it will also go to the correct element in the console

### CODE REVIEW

File: app/content/scripts/webviewPreload.js [hunk 1]
```diff
@@ -165,7 +165,11 @@
     })
   })
 
+  let submittedForms = []
   function onFormSubmit (form, formOrigin) {
+    if (submittedForms.includes(form)) {
+      return
+    }
```
Comment (bbondy) [inline app/content/scripts/webviewPreload.js]: I think this change wasn't intentional for this changeset

**Comment by bbondy** [comment]
Looks great otherwise, butt I think there was some unintentional changes in the diff relating to password saving.

**Comment by bobvanluijt** [comment]
Ooo, I think it was an older version... Later today I will re-commit with the newer version

**Comment by bobvanluijt** [comment]
Closing this one and I'll create a new one

**Comment by bobvanluijt** [comment]
Created a new request: https://github.com/brave/browser-laptop/pull/1150

**Pull request closed without merging**